README.md

```diff
@@ -152,9 +152,14 @@ If you're working on PR for this project, create a feature branch off of `main`.
 This repository follows the [Samvera Community Code of Conduct](https://samvera.atlassian.net/wiki/spaces/samvera/pages/405212316/Code+of+Conduct) and [language recommendations](https://github.com/samvera/maintenance/blob/main/templates/CONTRIBUTING.md#language).  Please ***do not*** create a branch called `master` for this repository or as part of your pull request; the branch will either need to be removed or renamed before it can be considered for inclusion in the code base and history of this repository.
 
 ## Compatibility
-
-- Ruby 2.5 or the latest 2.4 version is recommended. Later versions may also work.
-- Rails 5 is required. We recommend the latest Rails 5.2 release.
+This code works with the latest versions of:
+ - Rails 6.1 and 6.0.
+   - Rails 5 also works with Ruby versions older than 3.
+ - Ruby 3.0 (except with Rails 5), 2.6, and 2.5.
+ - The combination of Ruby 3 and Rails 6 comes with two caveats:
+   - your app will not be able to use `psych 4` (which ordinarily comes with 3.1). See https://bugs.ruby-lang.org/issues/17866 and https://stackoverflow.com/questions/71191685/visit-psych-nodes-alias-unknown-alias-default-psychbadalias/71192990#71192990 for an explanation. One workaround is to modify your app's `Gemfile` to hold back `psych`: `gem 'psych', '< 4'`.
+   - likewise, you want to add gem 'net-smtp', require: false to your Gemfile to get around the bug described in https://stackoverflow.com/questions/70500220/rails-7-ruby-3-1-loaderror-cannot-load-such-file-net-smtp .
+See also  `.circleci/config.yml` and `spec/test_app_templates/Gemfile.extra`.
```

Comment:
This section si good, but specifically is about Ruby 3.1 and Rails 6.1.

Reply:
Done!